README.md

```diff
@@ -15,7 +15,39 @@
 | NuGet Trends Worker | [![Build Status](https://dev.azure.com/nugettrends/nuget-trends/_apis/build/status/NuGetTrends%20-%20Worker?branchName=develop)](https://dev.azure.com/nugettrends/nuget-trends/_build/latest?definitionId=5&branchName=master) |
 | NuGet Trends SPA | - |
 
+## Developing
+
+We want to get a script to bootstrap the development environment but it's not here yet.
+That means now you'll need to:
+
+1. Run `docker-compose up` to get the services.
+
+And either:
+
+2. Restore a DB backup to get some data, like the packages and some download number.
+
+Or:
+
+2. Run _Entity Framework_ migrations to get the database schema in place.
+3. Run the _catalog importer_ from scratch.
```

Comment:
How do you run these importers? Is this automatically done by starting the `NuGetTrends.Scheduler` app?

Reply:
Yes. It's scheduled with Hangfire.
Good point! It's not clear.

Reply:
Added more details

Reply:
I've described you can go to the dashboard and run them manually. Let me know if it's clear now

Reply:
Perfect, thanks!